packages/react-core/src/components/Card/Card.tsx

```diff
@@ -79,6 +76,33 @@ export const Card: React.FunctionComponent<CardProps> = ({
     console.warn('Card: Cannot use isCompact with isLarge. Defaulting to isCompact');
     isLarge = false;
   }
+
+  if (isHoverable) {
+    // eslint-disable-next-line no-console
+    console.warn("Card: the 'isHoverable' prop is deprecated. Use isSelectable instead.");
+  }
+
+  const isRaised = selectableVariant === 'raised';
+
+  let selectableModifiers = '';
+  if (isSelectable || isHoverable) {
+    if (isRaised) {
+      if (isDisabled) {
+        selectableModifiers = css((isSelectable || isHoverable) && styles.modifiers.nonSelectableRaised);
+      } else {
+        selectableModifiers = css(
+          (isSelectable || isHoverable) && styles.modifiers.selectableRaised,
+          (isSelectable || isHoverable) && isSelected && styles.modifiers.selectedRaised
+        );
+      }
+    } else {
+      selectableModifiers = css(
+        (isSelectable || isHoverable) && styles.modifiers.selectable,
+        (isSelectable || isHoverable) && isSelected && styles.modifiers.selected
```

Comment:
Also, I don't think the repeated `isSelectable || isHoverable` conditionals here are needed as they are already nested within an `if` checking that conditional, so they will always be true. I could be missing something though.